assets/src/legacy/map.js

```diff
@@ -1366,33 +1368,11 @@ window.lizMap = function() {
 
                         // place children in the right div, if any
                         let relations = parentDiv.children('.container.popup_lizmap_dd').find(".popup_lizmap_dd_relation");
-                        relations.each((ind,relation) => {
-                            let referencingLayerId = $(relation).attr('data-referencing-layer-id');
-                            if (referencingLayerId) {
-                                let relLayerId = null;
-                                let referencingLayerConfig = lizMap.getLayerConfigById( referencingLayerId, lizMap.config.attributeLayers, 'layerId' );
-                                if (referencingLayerConfig && referencingLayerConfig[1]?.pivot == 'True' && config.relations.pivot && config.relations.pivot[referencingLayerId]) {
-                                    // relation is a pivot, search for "m" child layers
-                                    let mLayer = Object.keys(config.relations.pivot[referencingLayerId]).filter((k)=>{ return k !== layerId})
-                                    if (mLayer.length == 1) {
-                                        relLayerId = mLayer[0];
-                                    }
-                                } else {
-                                    // relation is 1:n, search for n layer
-                                    relLayerId = referencingLayerId;
-                                }
-
-                                if (relLayerId) {
-                                    let childLayer = childPopupElements.filter((child)=>{
-                                        let lName = child.attr("data-layername");
-                                        let lId = lName ? lizMap.config.layers?.[lName]?.id : '';
-                                        return relLayerId == lId;
-                                    })
-
-                                    if(childLayer.length == 1) {
-                                        $(relation).append(childLayer[0])
-                                    }
-                                }
+                        relations.each((ind, relation) => {
+                            let relationId = $(relation).attr('data-relation-id');
```

Comment:
```suggestion
                            let relationId = relation.dataset.relationId;
```
I think `relation` is an `HTMLElement` and `data-*` attribut is available through `dataset`